app/scripts/views/mixins/service-mixin.js

```diff
@@ -12,6 +12,10 @@ define(function (require, exports, module) {
   const $ = require('jquery');
 
   module.exports = {
+    afterRender () {
+      this.transformLinks();
```

Comment:
Just wondering, is there any performance cost to doing this in the views that weren't calling it explicitly before?